pkg/caveats/types/ipaddress.go

```diff
@@ -26,7 +25,7 @@ func MustParseIPAddress(ip string) IPAddress {
 	return ipAddress
 }
 
-var ipaddressCelType = types.NewTypeValue("IPAddress", traits.ReceiverType)
+var ipaddressCelType = cel.OpaqueType("IPAddress")
```

Comment:
Is this a breaking change? is it possible this would render existing serialized CEL AST incompatible?

Reply:
No, I don't believe so. It simply is the access semantics on the type